lib/src/components/NxDrawer/NxDrawer.scss

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @use '../../scss-shared/nx-container-helpers';
+ @use '../../scss-shared/nx-text-helpers';
+ 
+$-box-shadow-offset: 52px;
+
+.nx-drawer {
+  // Undo styles that chrome applies to <dialog> elements
+  border: initial;
+  height: initial;
+  left: initial;
+  margin: initial;
+  max-height: initial;
+  max-width: initial;
+  padding: initial;
+  width: initial;
+
+  // Prevent brief flash of scrollbar as tooltips disappear
+  overflow: hidden;
+
+  &::backdrop {
+    // Chrome tries to add its own subtle mask here which doubles up with our own
+    display: none;
+    pointer-events: auto;
+  }
+
+  background: transparent;
+  box-sizing: border-box;
+  height: 100%;
+  padding-left: $-box-shadow-offset;
+  position: fixed;
+  right: 0;
+  top: 0;
+  transform: translateX(100%);
+  width: 520px + $-box-shadow-offset;
+  z-index: 100;
+
+  &--with-global-header {
+    $global-header-height: 43px;
+    height: calc(100% - #{$global-header-height});
+    top: $global-header-height;
+  }
+  
+  &--narrow {
+    width: 348px + $-box-shadow-offset;
+  }
+
+  &__inner {
+    align-items: stretch;
+    background-color: var(--nx-swatch-white);
+    box-sizing: border-box;
+    display: flex;
```

Comment:
It might be simpler to use the `nx-container-sized` family of classes than to custom-manage the flex

Reply:
> It might be simpler to use the nx-container-sized family of classes than to custom-manage the flex

I meant `nx-viewport-sized`

Reply:
> > It might be simpler to use the nx-container-sized family of classes than to custom-manage the flex
> 
> I meant `nx-viewport-sized`

We attempted to implement it here, let us know if this is set up correctly: c784fa4f0e1221ae3e12ab5ccfe45d17298289ed

We just need to update the documentation regarding form.

Reply:
> We just need to update the documentation regarding form.

Actually I think we can mitigate the need for the caller to worry about that themselves by setting `display: contents` on `.nx-drawer__panel > .nx-form`.  Also, you have the wrong class on `NxDrawer.Content`.  It should have `.nx-viewport-sized__scrollable` and `.nx-scrollable`, not `.nx-viewport-sized__container`.

Reply:
> > We just need to update the documentation regarding form.
> 
> Actually I think we can mitigate the need for the caller to worry about that class themselves by setting `display: contents` on `.nx-drawer__panel > .nx-form`. Also, you have the wrong class on `NxDrawer.Content`. It should have `.nx-viewport-sized__scrollable` and `.nx-scrollable`, not `.nx-viewport-sized__container`.

Oh I see, thanks this is helpful:
Fixed it here: 2b34170714515fe29b72488e8a4e4a0c1d56f27b